### PR TITLE
Bugzilla 1540832: Updated the Enable Local Volumes section for K8s v1.9.0

### PR DESCRIPTION
**Pull request by ghost**
https://bugzilla.redhat.com/show_bug.cgi?id=1540832

@openshift/team-documentation PTAL. Rev history in the next comment.

### CODE REVIEW

**Comment by tmorriso-rh** [comment]
[rev_history]
|xref:../latest/install_config/configuring_local.adoc#install-config-configuring-local][Configuring for Local Volume]
|Updated the xref:../install_config/configuring_local.adoc#local-volume-enabling-local-volumes[Enable Local Volumes] section.
%

**Comment by tmorriso-rh** [comment]
@openshift/team-documentation PTAL. Rev history in the comment above.

File: install_config/configuring_local.adoc [hunk 1]
```diff
@@ -58,13 +58,19 @@ kubeletArguments:
      - PersistentLocalVolumes=true
 ----
 
+The `LocalPersistentVolumes` alpha feature also requires the `VolumeScheduling` alpha feature. This is a breaking change, and the following changes are required: 
+
+* The `VolumeScheduling` feature gate must also be enabled on kube-scheduler and kube-controller-manager components.
```
Comment (mburke5678) [inline install_config/configuring_local.adoc]: Do we have any info on how to enable the feature gate?

Comment (mburke5678) [reply install_config/configuring_local.adoc]: Do we have any info on how to enable the feature gate?

Comment (tmorriso-rh) [reply install_config/configuring_local.adoc]: I found this info on the Kubernetes reference site: https://kubernetes.io/docs/reference/feature-gates/

Should I add this link?

File: install_config/configuring_local.adoc [hunk 2]
```diff
@@ -58,13 +58,19 @@ kubeletArguments:
...
+* The `NoVolumeNodeConflict` predicate has been removed. For non-default schedulers, update your scheduler policy.
```
[the remark this answers is not in the document]
Comment (tmorriso-rh) [reply install_config/configuring_local.adoc]: I think the info will be included in the 3.9 release notes.

**Comment by tmorriso-rh** [comment]
Closing and continuing work in a new PR.

**Pull request closed without merging**